File: OKR.API/OKR.DTO/StatusStatistics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OKR.DTO
{
    public class StatusStatistics
    {
        public int noStatus {  get; set; }
        public int onTrack { get; set; }
        public int atRisk { get; set; }
        public int offTrack { get; set; }
        public int closed { get; set; }
        public int total { get; set; }
    }
}

File: OKR.UI/src/Models/StatusStatistics.ts
export class StatusStatistics {
    noStatus: number;
    onTrack: number;
    atRisk: number;
    offTrack: number;
    closed: number;
    total: number;

    constructor(
        noStatus: number = 0,
        onTrack: number = 0,
        atRisk: number = 0,
        offTrack: number = 0,
        closed: number = 0,
        total: number = 0
    ) {
        this.noStatus = noStatus;
        this.onTrack = onTrack;
        this.atRisk = atRisk;
        this.offTrack = offTrack;
        this.closed = closed;
        this.total = total;
    }
}